applications/StructuralMechanicsApplication/python_scripts/set_automated_initial_variable_process.py

```diff
@@ -0,0 +1,97 @@
+            "model_part_name"          : "please_specify_model_part_name",
+            "variable_name"            : "SPECIFY_VARIABLE_NAME",
+            "hole_generatrix_axis"     : [0.0,0.0,1.0],
+            "hole_generatrix_point"    : [0.0,0.0,0.0],
+            "hole_radius_offset"       : 0.0,
+            "initial_variable_table"     : {
+                        "name"             : "csv_table",
+                        "filename"         : "sample.csv",
+                        "delimiter"        : ",",
+                        "skiprows"         : 1,
+                        "first_column_id"  : 0,
+                        "second_column_id" : 1,
+                        "na_replace"       : 0.0
+                    }
+        }""")
+    process_settings = settings["Parameters"]
+    process_settings.ValidateAndAssignDefaults(default_settings)
+    computing_model_part = Model[process_settings["model_part_name"].GetString()]
+
+    default_table_id = KM.Parameters("""{
+    "table_id": 0
+    }""")
+    process_settings["initial_variable_table"].AddValue("table_id", default_table_id)
+
+    file_path = Path(process_settings["initial_variable_table"]["filename"].GetString())
+    layer_name = file_path.name.split("_")[0]
+    layer_list = [file for file in file_path.parent.iterdir() if file.name.split("_")[0] == layer_name]
+
+    if not len(layer_list):
+        ErrorMsg = "Tables of " + layer_name + " not found"
+        raise RuntimeError(ErrorMsg)
+    else:
+        component_list = []
+        table_id_list = []
+        out_of_range_component_list = []
+        for i in range (0, len(layer_list)):
+            component_number = int(layer_list[i].stem.split("_")[1][-1])
+            if  0 < component_number < 7:
+                component_list.append(component_number)
+                table_id = int(layer_name[-1] + str(component_number - 1))
+                table_id_list.append(table_id)
+                process_settings["initial_variable_table"]["filename"].SetString(layer_list[i].as_posix())
+                process_settings["initial_variable_table"]["table_id"].SetInt(table_id)
+                ReadCsvTableUtility(process_settings["initial_variable_table"]).Read(computing_model_part)
+            else:
+                out_of_range_component_list.append(component_number)
+                print (out_of_range_component_list)
+
+    raw_variable_name = process_settings["variable_name"].GetString()
+    variable_name = raw_variable_name.split("_")[0] + " " + raw_variable_name.split("_")[1].split("_")[0]
+
+    if len(out_of_range_component_list) != 0:
+        if len(out_of_range_component_list) == 1:
+            Logger.PrintInfo("::[WARNING]:: : SetAutomatedInitialVariableProcess ", variable_name.capitalize() + " component " + str(out_of_range_component_list[0]) + " of " + layer_name + " is out of range. The correspoding table will be negleted")
```

Comment:
```suggestion
            Logger.PrintWarning("SetAutomatedInitialVariableProcess ", f"{variable_name} component [out_of_range_component_list[0]} of {layer_name} is out of range. The corresponding table will be neglected")
```

please do the same for the other messages